other/lru_cache.py

```diff
@@ -114,11 +132,46 @@ def set(self, key: int, value: int) -> None:
             node.val = value
             self.list.add(node)
 
-    def has_key(self, key: int) -> bool:
+    def cache_info(self) -> str:
         '''
-        Checks if the input key is present in cache
+        Returns the details for the cache instance
+        [hits, misses, capacity, current size]
         '''
-        return key in self.cache
+
+        return f'CacheInfo(hits={self.hits}, misses={self.miss}, \
+capacity={self.capacity}, current size={self.num_keys})'
```

Comment:
```suggestion
        return (f'CacheInfo(hits={self.hits}, misses={self.miss},'
                f'capacity={self.capacity}, current size={self.num_keys})')
```
PEP8 is clear that backslashes should not be used in Python code because one whitespace character to the right of the backslash will break the script on a change that is invisible to the reader.